internal/actions/update.go

```diff
@@ -25,11 +26,13 @@ func Update(client container.Client, params types.UpdateParams) error {
 		return err
 	}
 
-	for i, container := range containers {
-		stale, err := client.IsContainerStale(container)
+	for i, targetContainer := range containers {
+		stale, err := client.IsContainerStale(targetContainer)
+		if stale && !params.NoRestart && !params.MonitorOnly && !targetContainer.HasImageInfo() {
+			err = errors.New("container does not have any available image info and cannot be automatically restarted")
```

Comment:
Mind changing this to an `Errorf` and include the container name like on line 35?

Reply:
Its handled just beneath and combined with the other situation where updating is skipped. I did shorten the message though, as it becomes quite verbose.